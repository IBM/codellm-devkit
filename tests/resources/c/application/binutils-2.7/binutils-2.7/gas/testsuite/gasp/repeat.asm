
	.AREPEAT	10
	TEN
	.AREPEAT	2	
	TWENTY
	.AENDR
	.AENDR

	.AREPEAT	3
	ROTCL	R2
	DIV1	R0,R1
	.AENDR

	.END
