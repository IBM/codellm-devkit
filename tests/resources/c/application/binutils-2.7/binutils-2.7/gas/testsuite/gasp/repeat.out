!

!	.AREPEAT	10
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	9
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	8
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	7
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	6
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	5
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	4
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	3
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	2
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	.AENDR
!	TEN
	TEN
!	.AREPEAT	2	
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!	.AREPEAT	1
!	TWENTY
!	.AENDR
!	TWENTY
	TWENTY
!

!	.AREPEAT	3
!	ROTCL	R2
!	DIV1	R0,R1
!	.AENDR
!	ROTCL	R2
	ROTCL	R2
!	DIV1	R0,R1
	DIV1	R0,R1
!	.AREPEAT	2
!	ROTCL	R2
!	DIV1	R0,R1
!	.AENDR
!	ROTCL	R2
	ROTCL	R2
!	DIV1	R0,R1
	DIV1	R0,R1
!	.AREPEAT	1
!	ROTCL	R2
!	DIV1	R0,R1
!	.AENDR
!	ROTCL	R2
	ROTCL	R2
!	DIV1	R0,R1
	DIV1	R0,R1
!

!	.END
